routersploit/modules/exploits/cameras/multi/netwave_IP_camera.py

```diff
@@ -0,0 +1,78 @@
+from routersploit import (
+    exploits,
+    print_error,
+    print_success,
+    print_info,
+    http_request,

```

Comment:
Add mute (decorator for check method),
Add validators (for validating parameters)